airbyte-integrations/connectors/source-instagram/metadata.yaml

```diff
@@ -19,6 +19,13 @@ data:
     oss:
       enabled: true
   releaseStage: generally_available
+  releases:
+    breakingChanges:
+      2.0.0:
+        message:
+          This release introduces a default primary key for the streams UserLifetimeInsights and UserInsights.
+          Additionally, the format of timestamp fields has been updated to include timezone information.
```

Comment:
Additionally, the format of timestamp fields has been updated in the Media and Stories streams to include timezone information.

Reply:
Fixed